pkg/query-service/app/clickhouseReader/reader.go

```diff
@@ -762,11 +762,16 @@ func (r *ClickHouseReader) GetServices(ctx context.Context, queryParams *model.G
 				args...,
 			).ScanStruct(&serviceItem)
 
+			if serviceItem.NumCalls == 0 {
+				return
+			}
+
 			if err != nil {
 				zap.S().Error("Error in processing sql query: ", err)
 				return
 			}
 
+			args, errStatus = buildQueryWithTagParams(ctx, queryParams.Tags, &errorQuery, args)
```

Comment:
@srikanthccv this is related to the scope of PR or should have been there already?

Reply:
This should have been there already similar to line 754. Couldn't summarize all changes in title. This fixes the known resource attributes filter bug, does not showing zero values, and fixes the bug with error when filters are applied.